### PR TITLE
Added PHP CS Fixer configuration

### PR DESCRIPTION
**Pull request by bdunogier**
Title says it all.

### CODE REVIEW

**Comment by bdunogier** [comment]
@andrerom how do we enable ezrobot for this ?

File: .php_cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?php
+
+if (class_exists('Symfony\CS\Config\Config')) {
+    // PHP-CS-Fixer 1.x syntax (deprecated)
```
Comment (andrerom) [inline .php_cs]: don't need this, we only support one version and that is 2.x. the if cases where just needed during transition from 1.x ot 2.x

**Review by andrerom — CHANGES_REQUESTED** [review]

**Comment by andrerom** [comment]
> @andrerom how do we enable ezrobot for this ?

afaik you create a new job on jenkins, @vidarl might be able to show you how :)

**Review by andrerom — APPROVED** [review]

**Comment by andrerom** [comment]
+1 assuming it passes ;)

**Comment by bdunogier** [comment]
Okay, php-cs-fixer passed, merging

**Pull request merged**